### ISSUE TITLE
Timeline doesn't showing commits to repository after any tag is created

### ISSUE THREAD

**Issue by VitSimon** [go-gitea/gitea#17503]
### Gitea Version

1.14.3

### Git Version

2.30.2

### Operating System

Ubuntu 5.4.0-87-generic / 18.04.6 LTS

### How are you running Gitea?

Git: 2.30.2, Wire Protocol Version 2 Enabled
Gitea docker image: gitea/gitea:1.14.3
Database: MariaDB - docker image mariadb:10.6.2-focal
Docker: 20.10.9, build c2ea9bc

### Database

MySQL

### Can you reproduce the bug on the Gitea demo site?

No

### Description

Hello,

I noticed when I created a tag on some the timeline is not anymore updated for my another commits afterwards.
I couldnt reproduce it on your test page ( tried here: https://try.gitea.io/VitSimon/Test2 ) . 
My version is a bit older than on try page - 1.14.3.

Every time I commited with client GitExtensions and git-cola over URI path. 
Using Windows and Linux machine over repo alternately. 
1st tag has been created via client and pushed to repo from over 200 commits in past, rest of tags were created from Gitea web UI.

### Screenshots

![GiteaTimeLine](https://user-images.githubusercontent.com/89648814/139585973-e26fa92a-1e0c-4dd0-a0f5-76d771c08d63.png)
![GiteaTimeLine2](https://user-images.githubusercontent.com/89648814/139585278-06c2e5a7-3df9-4056-956e-fc48da602170.png)

**Comment by zeripath** [comment]
You're gonna have to upgrade to 1.15.6 and see if you can reproduce on 1.15.

1.14 is no longer supported.

**Comment by VitSimon** [comment]
@zeripath : Thanks for reply. Tried to upgrade and now my version is: 1.15.6:

![Version](https://user-images.githubusercontent.com/89648814/141183367-8c5aa7a7-311f-458e-89f1-03326ecb8c5c.png)

Unfortunately my state is the same as when 1stly asked:

![GiteaTimeLine](https://user-images.githubusercontent.com/89648814/141183467-84a79b28-fb2e-4bc1-8ccf-1bcb705c376b.png)

![GiteaTimeLine2](https://user-images.githubusercontent.com/89648814/141183477-4119c226-2fb7-43e4-86d9-c62925dc2df4.png)

Possibly can be interesting: my repository has been created by git svn init + fetch (it has been SVN before).
Until I created 1st tag, this functionality worked well for me.
